mk/e2e.new.mk

```diff
@@ -38,6 +38,7 @@ K8S_CLUSTER_TOOL=k3d
 E2E_ENV_VARS += KUMA_K8S_TYPE=k3d
 else
 K8S_CLUSTER_TOOL=kind
+GINKGO_E2E_TEST_FLAGS += --label-filter="!kind-not-supported"
```

Comment:
isn't it `!k3d-not-supported`?

Reply:
no, the only cluster that supports this is `k3d` so if we detect `kind` we should filter out the tests with label `kind-not-supported`

Docs:
![image](https://user-images.githubusercontent.com/2753650/181739761-3e7ce224-0007-4c46-982c-0f4e4f6309d8.png)